src/DssSpell.sol

```diff
@@ -18,69 +18,20 @@ pragma solidity 0.8.16;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
-import "dss-interfaces/dss/IlkRegistryAbstract.sol";
-import "dss-interfaces/ERC/GemAbstract.sol";
-
-interface GemLike {
-    function transfer(address, uint256) external returns (bool);
-}
-
-interface VatLike {
-    function ilks(bytes32) external view returns (uint256 Art, uint256 rate, uint256 spot, uint256 line, uint256 dust);
-}
-
-interface RwaLiquidationLike {
-    function ilks(bytes32) external view returns (string memory doc, address pip, uint48 tau, uint48 toc);
-    function init(bytes32 ilk, uint256 val, string memory doc, uint48 tau) external;
-    function bump(bytes32 ilk, uint256 val) external;
-}
-
-interface ACLManagerLike {
-    function addPoolAdmin(address admin) external;
-}
-
-interface ProxyLike {
-    function exec(address target, bytes calldata args) external payable returns (bytes memory out);
-}
-
-interface Initializable {
-    function init(bytes32 ilk) external;
-}
-
-interface RwaUrnLike {
-    function hope(address usr) external;
-    function nope(address usr) external;
-    function lock(uint256 wad) external;
-    function draw(uint256 wad) external;
-}
-
-interface RwaInputConduitLike {
-    function mate(address usr) external;
-    function hate(address usr) external;
-    function file(bytes32 what, address data) external;
-}
-
-interface RwaOutputConduitLike {
-    function file(bytes32 what, address data) external;
-    function hope(address usr) external;
-    function nope(address usr) external;
-    function mate(address usr) external;
-    function hate(address usr) external;
-    function kiss(address who) external;
-    function pick(address who) external;
-    function push() external;
-}
 
 contract DssSpellAction is DssAction {
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/c55480785534ee1f75bda65afb9761ffaebe072f/governance/votes/Executive%20vote%20-%20June%2014%2C%202023.md' -q -O - 2>/dev/null)"
+    // Hash: cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/4b8f4ba042910b2fd8d27ccd06a15988ff23c113/governance/votes/Executive%20vote%20-%20June%2016%2C%202023.md' -q -O - 2>/dev/null)"
```

Comment:
Exec hash matches on my end, though the commit hash per the history tab is `2f00c26105df8ca8a097bd9d94cb1f691c6c2740` (equivalent comment).

Requesting this commit hash is updated to match what appears at the top of the page [here](https://github.com/makerdao/community/commits/master/governance/votes/Executive%20vote%20-%20June%2016%2C%202023.md)

Reply:
Given that there is no official rule in the checklist for this, I am OK to resolve this issue as it is non-functional and not security related. Of course, I would be more comfortable approving this spell if the commit hash matched the way PECU has historically fetched it, but as mentioned elsewhere, we are in a rush and this is a minor issue relative to others raised. 
Resolving for now.